### ISSUE TITLE
How to use BertModel ?

### ISSUE THREAD

**Issue by RoderickGu** [huggingface/transformers#813]
I want to use bert-crf in my NER task. But this github only provide softmax as the classifier, I decided to write my own crf. But I am not sure how to use it. Here is an example. Please correct me if I am wrong.

sentence: Here is some text to encode
input: torch.tensor([tokenizer.encode("[CLS]" + "Here is some text to encode" + "[SEP]")]), which is 
tensor([[  101,  3446,  1110,  1199,  3087,  1106,  4035, 13775,   102]])

output shape: [1, 9, 768],

Becasue "encode" is devided into two pieceword.
Then I select output[1, 2, 3, 4, 5, 6] from output[0,1,2,3,4,5,6,7,8]to get the crf_input that is of shape [1, 6, 768]. 

That is how I think it should work. Any suggestion will be appreciated.

**Comment by weizhepei** [comment]
This issue has been discussed at [#64](https://github.com/huggingface/pytorch-transformers/issues/64#issuecomment-443703063).

**Comment by chnsh** [comment]
Hi - I tried to implement it, you can have a look at my implementation here: https://github.com/chnsh/BERT-NER-CoNLL, hope that helps

**Comment by hasansalimkanmaz** [comment]
@chnsh  I don't see any crf layer on your github repo.

**Comment by hasansalimkanmaz** [comment]
@RoderickGu Were you able to implement bert-crf?
